genai-perf/tests/test_cli.py

```diff
@@ -568,70 +566,6 @@ def test_unrecognized_arg(self, monkeypatch, capsys):
                 ],
                 "The --output-tokens-mean-deterministic option is only supported with the Triton and TensorRT-LLM Engine service-kind",
             ),
-            (
```

Comment:
Why are these unnecessary now?

Reply:
These checks were moved to the converters, since they're format-specific. To allow GAP to be more extensible (and logically organized), format-specific checks have been moved to a check_config function implemented in the output format converters. See [this one](https://github.com/triton-inference-server/perf_analyzer/pull/142/files#diff-d25d2d07884eb6a98fb2c89b98a26c4669d89586646ceb103fe268a91deaac67R36) as an example.